src/components/pages/landing-de.tsx

```diff
@@ -1,203 +1,250 @@
-      <SubjectsSection>
+      <section>
         <LandingSubjects data={subjectsData} />
-      </SubjectsSection>
+      </section>
 
-      <AboutSection>
+      <section className="mt-12 flex flex-col sm:flex-row">
         <LandingAbout />
-      </AboutSection>
-
-      <Section>
-        <StyledH2>Serlo.org ist die Wikipedia fürs Lernen</StyledH2>
-        <IconStyle>
-          <ParticipateSVG />
-        </IconStyle>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Wir suchen Lehrkräfte mit Begeisterung für ihr Fach. Werden Sie
-            Autor*in auf serlo.org, erstellen Sie <b>neue Inhalte</b> und helfen
-            Sie uns, die <b>Qualität</b> der Lernplattform zu sichern.
-          </p>
-          <Button href="/community">
-            Zur Startseite für Autor*innen{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Wir suchen neue hauptamtliche und ehrenamtliche Teammitglieder für
-            die Bereiche <b>Softwareentwicklung</b>, <b>Redaktion</b> und{' '}
-            <b>NGO-Management</b>.
-          </p>
-          <Button href="/jobs">
-            Jobs und Engagement{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-      </Section>
-
-      <PrinciplesSection>
-        <PrinciplesGraphic strings={landingStrings} />
-      </PrinciplesSection>
-
-      <Section>
-        <StyledH2>Werden Sie Teil unserer Bewegung für freie Bildung</StyledH2>
-        <IconStyle>
-          <DonateSVG />
-        </IconStyle>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Bildung gehört uns allen! Werden Sie Mitglied in unserer
-            Organisation Serlo Education e.V. und so zu Mitherausgeber*in der
-            freien Lernplattform.
-          </p>
-          <Button href="/beitreten">
-            Mitglied werden{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Softwareentwicklung und Lerninhalte erstellen kostet Geld. Wir
-            freuen uns sehr, wenn Sie Serlo mit einer Spende unterstützen.
-          </p>
-          <Button href="/spenden">
-            Spenden <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-      </Section>
-    </>
+      </section>
+
+      <section>
+        <h3
+          className={clsx(
+            'text-center text-4xl text-truegray-700 font-bold',
+            'leading-cozy tracking-tight',
+            'max-w-2xl mt-40 mx-auto relative z-10'
+          )}
+        >
+          Wir sind eine große, ehrenamtliche Community und gestalten Serlo
+          <p className="text-brand italic font-handwritten">gemeinsam.</p>
```

Comment:
`gemeinsam` is too small. 
I suggest `font size: 2.7rem;` and maybe `margin-top: -4rem;´ on the background to match the design more closely.

Reply:
I have increased the font-size and it looks fine (for reference: the design is using 2.5rem / 3rem)

Reply:
Looks good 👍